editor/modes/visual-editor/style.scss

```diff
@@ -446,12 +446,16 @@
 .editor-visual-editor__sibling-inserter {
 	z-index: z-index( '.editor-visual-editor__sibling-inserter' );
 	position: relative;
-	max-width: $visual-editor-max-width + ( 2 * $block-mover-padding-visible );
+	max-width: $visual-editor-max-width + ( 2 * $block-mover-padding-visible ) - ( 2 * $block-padding );
```

Comment:
Wondering how you came to this change, if it's only applying on sub-"small" screens and `max-width` would already be a larger value than what we'd expect (`636 + ( 2 * 32 ) - ( 2 * 14 ) = 672 > 600`).

Reply:
The idea was to mimic the `.editor-visual-editor__block` width minus its padding because the block mover and the block menu are shown in this padding space

Reply:
Hmm, okay, it's strange, since it's mathematically impossible for the width to ever exceed this value anyways in the sub-small viewport range, but I guess since `$break-small` could potentially be revised separate from these other variables, it could be wise to keep them disconnected.